lib/main.ts

```diff
@@ -773,7 +836,7 @@ export class Transpiler {
         this.emit(`'''${this.escapeTextForTemplateString(node)}'''`);
         break;
       case ts.SyntaxKind.TemplateMiddle:
-        this.result += this.escapeTextForTemplateString(node);
+        this.emitNoSpace(this.escapeTextForTemplateString(node));
```

Comment:
these changes hint at a refactoring: maybe the result should be a private field in another class which represents the generated code. that class could encapsulate the responsibility of tracking the line and column